packages/devextreme/js/ui/tabs.js

```diff
@@ -123,14 +131,23 @@ const Tabs = CollectionWidget.inherit({
         ]);
     },
 
+    _setBaseElementClasses() {
+        const isVertical = this.option('orientation') === ORIENTATION.vertical;
+        const orientationClass = isVertical ? TABS_VERTICAL_CLASS : TABS_HORIZONTAL_CLASS;
+
+        const elementClasses = [
+            TABS_CLASS,
+            orientationClass,
+        ].join(' ');
+
+        this.$element().addClass(elementClasses);
+    },
+
     _init: function() {
         this.callBase();
-
         this.setAria('role', 'tablist');
-
-        this.$element().addClass(TABS_CLASS);
+        this._setBaseElementClasses();
```

Comment:
I'd rather to add dx-tabs class here
```
this.$element().addClass(TABS_CLASS)
```
The logic with toggling orientation classes to extract to separate method as follows
``` 
toggleOrientationClass(orientation) { ...}
```
and use it inside optionChanged method and other places where it needs

Reply:
Nice point, it really looks better) Thanks!